### DISCUSSION TITLE
Help please with liveupdate and listboxes

### DISCUSSION THREAD

**Discussion by WinterMUD** [cyrusfirheir/cycy-wrote-custom-macros#11] [Q&A] [answered]
Continuftion of https://github.com/cyrusfirheir/cycy-wrote-custom-macros/issues/9

How to do several such pairs of listboxes on one passage? In other words I need several such liveblocks on one page.

**Comment by cyrusfirheir** [comment, marked as answer]
I don't see how multiple of the same thing would be a problem. You can just replicate that structure for the rest.

**Reply by WinterMUD** [reply]
Only first pair works as needed. Other updates only if update page or update first pair.

**Reply by cyrusfirheir** [reply]
Something like this should work:

```html
<span class="list-controller">
  <<listbox "$one">>
    <!-- first controller -->>
  <</listbox>>
</span>

<<liveblock>>
  <!-- stuff for second listbox -->
<</liveblock>>


<span class="list-controller">
  <<listbox "$two">>
    <!-- second controller -->>
  <</listbox>>
</span>

<<liveblock>>
  <!-- stuff for second listbox -->
<</liveblock>>


<<done>>
  <<run $(".list-controller").on("input", function() {
    $(document).trigger(":liveupdate");
  })>>
<</done>>
```

**Reply by WinterMUD** [reply]
It seems works now. Only I change "input" on "change". Thanks very much.
